src/lib/ns/user/user.c

```diff
@@ -39,11 +39,32 @@
 #include "lib/config_parser.h"
 #include "lib/privilege.h"
 
-static int enabled = -1;
-
 int singularity_ns_user_enabled(void) {
-    singularity_message(DEBUG, "Checking user namespace enabled: %d\n", enabled);
-    return(enabled);
+    singularity_message(DEBUG, "Checking if user namespace is enabled.\n");
```

Comment:
This does not take into consideration what I was initially going for with this function. The original idea was not to return if user namespace is possible, but to return if it is currently running. There are functions in the source that are expecting the previous behavior of this function (e.g. rootfs/dir.c and fork.c possibly others).

There is also some duplicate funcationatly presently in singularity_priv_userns_ready() and singularity_priv_userns_enabled(). I see pros and cons to using singularity_priv_userns_* as well as singularity_ns_user_*. If you are suggesting we use singularity_priv_userns_*, then we will need to go back and make sure we have changed all of the other calls to singularity_ns_user_enabled() to be either expecting this new return model, or switch them to use singularity_priv_userns_enabled().